### PR TITLE
Add Stripe functionality and augment Navbar functionality

### PR DESCRIPTION
**Pull request by jeffgore00**
### Assignee Tasks

- [ ] added unit tests (or none needed)
- [ ] written relevant docs (or none needed)
- [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

*Your PR Notes Here*

### CODE REVIEW

File: client/components/checkout.js [hunk 1]
```diff
@@ -1,68 +1,83 @@
...
-            </button>
-        </form>
+        <button
+          type="submit"
+          ref={input => { inputElement = input; }}
+        />
+      </form>
+      <h3>Payment Information:</h3>
+      <StripeCheckout
+        stripeKey="pk_test_vt2tbGVU5eFuxMOOljFfZHew"
```
Comment (fma1) [inline client/components/checkout.js]: So I'm going to add the secret key to Heroku's config vars. I don't think it's good practice to put the secret in the code.

Comment (fma1) [reply client/components/checkout.js]: https://hackernoon.com/how-to-use-environment-variables-keep-your-secret-keys-safe-secure-8b1a7877d69c

Comment (RedPineapple) [reply client/components/checkout.js]: I think that the normal thing to do is to have it in the .env file, and then you can access it with something like:
const stripe = require('stripe')(process.env.STRIPE_SECRET);

Comment (RedPineapple) [reply client/components/checkout.js]: ninja'd... rip

Comment (fma1) [reply client/components/checkout.js]: I'm dumb this is the public key.

**Pull request merged**